### PR TITLE
Feature --> Develop

### PR DESCRIPTION
**Pull request by tkachuk09**

### CODE REVIEW

**Review by kteneta — CHANGES_REQUESTED** [review]

File: provider.tf [hunk 1]
```diff
@@ -0,0 +1,4 @@
+provider "aws" {
+      region = var.aws_region
+      shared_credentials_file = "/vlad/.aws/credentials"
```
[the remark this answers is not in the document]
Comment (tkachuk09) [reply provider.tf:3]: got it

File: resourse.tf [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  ami           = "ami-0194c3e07668a7e36"
+  instance_type = "t2.micro"
+
+  subnet_id               = aws_subnet.public_subnet.id
+  key_name                = aws_key_pair.london_region_key_pair.id
+}
+
+resource "aws_key_pair" "london_region_key_pair" {
+  key_name   = "london_region_key_pair"
+  public_key = file("/home/vlad/Desktop/london_region_key_pair.pub")
```
[the remark this answers is not in the document]
Comment (tkachuk09) [reply resourse.tf:11]: +

[the remark this answers is not in the document]
Comment (tkachuk09) [reply resourse.tf:11]: +

File: vpc.tf [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    cidr_block = var.route_table_cidr
+    gateway_id = aws_internet_gateway.gateway.id
+  }
+
+  tags       = {
+    Name     = "route_table"
+  }
+}
+# Associate Public Subnet to "Public Route Table"
+resource "aws_route_table_association" "public_subnet_route_table_association" {
```
[the remark this answers is not in the document]
Comment (tkachuk09) [reply vpc.tf:44]: will be done soon

File: vpc.tf [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+
+# Associate Private Subnet to "Private Route Table"
+resource "aws_route_table_association" "private_subnet_route_table_association" {
+  subnet_id      = aws_subnet.private_subnet.id
+  route_table_id = aws_route_table.private_route_table.id
+}
+
+# Create NAT gateway
+
+resource "aws_nat_gateway" "gateway" {
```
Comment (kteneta) [inline vpc.tf:81]: you don't seem to use this nat resource anywhere

File: vpc.tf [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+    Name    = "private_subnet"
+  }
+}
+# Create Route Table for private subnet
+resource "aws_route_table" "private_route_table" {
+  vpc_id       = aws_vpc.vpc.id
+
+  route {
+    cidr_block = var.route_table_cidr
+    gateway_id = aws_internet_gateway.gateway.id
```
Comment (kteneta) [inline vpc.tf:65]: i am not sure if it's fine that you're adding igw to private route table

**Pull request merged**